docs/sources/developers/plugins/development-with-local-grafana.md

```diff
@@ -74,25 +78,27 @@ Finally start your plugin in development mode. Go to your plugin root directory
    yarn start
    ```
 
-   2.2 Or with docker-compose, in your plugin directory, run:
+   b. Or, with docker-compose in your plugin directory, run:
 
    ```bash
    docker-compose up
    ```
 
-After this, you should be able to see your plugin listed in Grafana and test your changes. Note that any change in the fronted will require you to refresh your browser while changes in the backend may require to rebuild your plugin binaries and reload the plugin (`mage && mage reloadPlugin` for local development or `docker-compose up` again if you are using docker-compose).
+After this, you should be able to see your plugin listed in Grafana, and then you can test your changes. 
+
+If you make a change in the fronted, you must refresh your browser. However, changes in the backend may require that you rebuild your plugin binaries and reload the plugin (`mage && mage reloadPlugin` for local development, or run `docker-compose up` again if you are using docker-compose).
 
 ## Run your backend plugin with a debugger
 
 > Note: The following method only works with a local Grafana instance and currently doesn't work with Docker.
 
-You can run a backend plugin and attach a debugger to it, which allows you to set breakpoints and debug your backend plugin directly from your IDE of choice.
+Running a backend plugin with a debugger is a feature that supports Visual Studio Code and GoLand out of the box, but it can also work with any other IDE or debugger.
```

Comment:
```suggestion
Running a backend plugin with a debugger is supported in Visual Studio Code and GoLand out of the box, but it can also work with any other IDE or debugger.
```